presto-native-execution/CMakeLists.txt

```diff
@@ -165,6 +165,7 @@ include_directories(.)
 include_directories(velox)
 include_directories(velox/velox/external/xxhash)
 include_directories(${VELOX_ROOT})
+include_directories(${CMAKE_BINARY_DIR})
```

Comment:
This is to fix the proto path issue triggered by this PR.

'velox/velox/dwio/dwrf/proto/dwrf_proto.pb.h' file not found